### ISSUE TITLE
why the validation set data is not used in NavSim v2

### ISSUE THREAD

**Issue by wangxiyang2022** [valeoai/DrivoR#39]
Hello, this is excellent work—thank you for sharing it. 

I was wondering if you could kindly explain why the validation set data is not used in NavSim v2?

<img width="876" height="515" alt="Image" src="https://github.com/user-attachments/assets/0129ef3f-c389-4479-9ced-195dca66c07d" />

**Comment by yihongXU** [comment]
Hi, 

Before using Simscale data, we observed no improvement on Nav2 (especially the static images), so we discarded the val from training.

best

**Comment by wangxiyang2022** [comment]
Thanks for your reply. 

Does this mean adding the 18k validation set brings no improvement on Nav2?

Best regards,

**Comment by yihongXU** [comment]
We haven't tried with simscale data but comparing train, or trainval for nav2, yes:

>  On NAVSIM-v2 (warmup-two-stage), we observe an opposite trend, with EPDMS decreasing with the number of training epochs
as well as training using the competition split.

**Comment by wangxiyang2022** [comment]
Thanks!
